packages/contractkit/package.json

```diff
@@ -20,7 +20,7 @@
     "clean:all": "yarn clean && rm -rf src/generated",
     "build:gen": "yarn --cwd ../protocol build",
     "prepublishOnly": "yarn build:gen && yarn build",
-    "test:prepare": "yarn --cwd ../protocol devchain generate .devchain --migration_override src/test-utils/migration-override.json",
+    "test:prepare": "yarn --cwd ../protocol devchain generate .devchain --migration_override ../dev-utils/src/migration-override.json",
```

Comment:
@mcortesi Should this be `pretest` instead of `test:prepare`?

Reply:
I believe this command is expected to run before every test which is why it is not named `pretest`